### DISCUSSION TITLE
<meta tags>

### DISCUSSION THREAD

**Discussion by JesseOSullivan** [decaporg/decap-cms#6813] [Q&A]
How would I go about implementing meta tags automatically for each blog post. I am currently using the hugo template and it seems this is not implemented by default.

I've tried to search for docs about it in decap however have run dry

**Comment by martinjagodic** [comment]
You will have to do that with Hugo, not with CMS. 

I would add something like this to `head` the baseof layout:

```html
<meta property="og:title" content="{{ .Title }}">
<meta property="og:description" content="{{ .Description }}">
```
